COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
@eduarrrd send me your code, I will remove that duty from you and commit under my own name :)
Indeed. C'mon. You are person, not company.